sycl/include/CL/sycl/detail/pi.hpp

```diff
@@ -206,6 +206,16 @@ template <> inline void print<>(pi_image_offset off) {
             << off->z << std::endl;
 }
 
+template <> inline void print<>(const pi_image_desc *desc) {
+  std::cout << "image_desc w/h/d : " << desc->image_width << " / "
```

Comment:
Could you remove the multiple `desc->` from the output?

Reply:
I'm not sure I follow you.   The string "desc->" does not appear in the output string ever.  The final output string looks like this:
`image desc w/h/d : 200/30/3 -- arrSz/row/slice : 0/100/3000 -- num_mip_lvls/num_smpls/image_type : 0/0/4388`

Did I misunderstand?

Reply:
I'm not sure I'm following you.  The string "desc->" never appears in the final output.  The outputted string looks like this:
`image_desc w/h/d : 200/40/5 -- arrSz/row/slice : 0/800/36000  -- num_mip_lvls/num_smpls/image_type : 0/0/4388`

Apologies if I misunderstand your comment.

Reply:
Sorry, I misread the code.